### PR TITLE
fix test nil panic error.

### PR DESCRIPTION
**Pull request by zonghaishang**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/configmanager/configmanager.go [hunk 1]
```diff
@@ -213,17 +213,24 @@ func addOrUpdateRouterConfig(routerConfig *v2.RouterConfiguration) bool {
...
-	// support only one server
-	routers := config.Servers[0].Routers
-	for idx, rt := range routers {
-		if rt.RouterConfigName == routerConfig.RouterConfigName {
-			config.Servers[0].Routers[idx] = routerConfig
-			return true
+
+	// easy for test (config.Servers maybe equals nil),
+	// for example, a registry push does not trigger the dump store
+	if config.Servers != nil {
```
Comment (cch123) [inline pkg/configmanager/configmanager.go:219]: if config.Servers != nil {
     return false
}

... other logic

Comment (zonghaishang) [reply pkg/configmanager/configmanager.go]: ok , fixed it.

**Comment by cch123** [comment]
LGTM

**Review by nejisama — APPROVED** [review]
LGTM
